plugin.json

```diff
@@ -19,73 +19,73 @@
+                "display_name": "GitHub OAuth Client Secret:",
                 "type": "text",
                 "help_text": "The client secret for the OAuth app registered with GitHub."
             },
             {
                 "key": "WebhookSecret",
-                "display_name": "Webhook Secret",
+                "display_name": "Webhook Secret:",
                 "type": "generated",
                 "help_text": "The webhook secret set in GitHub."
             },
             {
                 "key": "EncryptionKey",
-                "display_name": "At Rest Encryption Key",
+                "display_name": "At Rest Encryption Key:",
                 "type": "generated",
                 "help_text": "The AES encryption key used to encrypt stored access tokens."
             },
             {
                 "key": "GithubOrg",
-                "display_name": "GitHub Organization",
+                "display_name": "GitHub Organization:",
                 "type": "text",
                 "help_text": "(Optional) Set to lock the plugin to a single GitHub organization."
             },
             {
                 "key": "EnterpriseBaseURL",
-                "display_name": "Enterprise Base URL",
+                "display_name": "Enterprise Base URL:",
                 "type": "text",
                 "help_text": "(Optional) The base URL for using the plugin with a GitHub Enterprise installation. Example: https://github.example.com"
             },
             {
                 "key": "EnterpriseUploadURL",
-                "display_name": "Enterprise Upload URL",
+                "display_name": "Enterprise Upload URL:",
                 "type": "text",
                 "help_text": "(Optional) The upload URL for using the plugin with a GitHub Enterprise installation. This is often the same as your Base URL."
             },
             {
                 "key": "EnablePrivateRepo",
-                "display_name": "Enable Private Repositories",
+                "display_name": "Enable Private Repositories:",
                 "type": "bool",
                 "help_text": "(Optional) Allow the plugin to work with private repositories. When enabled, existing users must reconnect their accounts to gain access to private repositories. Affected users will be notified by the plugin once private repositories are enabled."
             },
             {
                 "key": "EnableCodePreview",
-                "display_name": "Enable Code Previews",
+                "display_name": "Enable Code Previews:",
                 "type": "dropdown",
                 "help_text": "Allow the plugin to expand permalinks to GitHub files with an actual preview of the linked file.",
                 "default": "public",
                 "options": [
                     {
-                        "display_name": "Enable for public repositories",
+                        "display_name": "Enable for public repositories:",
```

Comment:
That seems odd. Usually drop down options don't end with a punctuation. Can you please double check if that is intended?

Reply:
No, you're correct, that was my bad.